src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -181,7 +181,10 @@ class ActivityFeed extends React.Component<Props, State> {
 
     componentWillReceiveProps(nextProps: any): void {
         const { comments, tasks, versions } = nextProps;
-        this.sortFeedItems(comments, tasks, versions);
+        const { feedItems } = this.state;
+        if (!feedItems) {
```

Comment:
Would there be any problems for checking against an empty array vs. undefined?

Reply:
Now checking length of array :)
I can remove the array fallback from `shouldShowEmptyState()` above